letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
In that case, be sure to leave a big, clear comment above your hard-coded list explaining what it's for so adders of new bootstrap scripts won't be tempted to add their script to the list just because the rest of them seemed to be there. :-)